x-pack/plugins/canvas/public/components/page_manager/page_manager.scss

```diff
@@ -88,6 +88,7 @@
     @include euiBottomShadowSmall;
     position: relative;
     overflow: hidden;
+    color: $euiColorFullShade;
```

Comment:
We can use our default EUI text color variable ... 
'$euiTextColor'

Thanks!

Reply:
You got it!